src/ray/gcs/gcs_client.h

```diff
@@ -87,6 +68,13 @@ class GcsClient : public std::enable_shared_from_this<GcsClient> {
     return *actor_accessor_;
   }
 
+  /// Get JobInfoAccessor for reading or writing or subscribing to
```

Comment:
```suggestion
  /// Get the sub-interface for accessing job information in GCS.
```
So it's aligned with `JobInfoAccessor`'s documentation.